### ISSUE TITLE
Additional Service Support Columns

### ISSUE THREAD

**Issue by PatMyron** [SummitRoute/aws_research#9]
[CloudFormation](https://github.com/SummitRoute/aws_research/pull/13) / [Tagging](https://github.com/SummitRoute/aws_research/pull/14) / Autoscaling / Backup

**Comment by 0xdabbad00** [comment]
It'd be great to receive a PR for these. Tagging and CloudFormation make sense, but I don't think autoscaling or backup make sense as a concept for all services. For example S3 doesn't have a concept of autoscaling, and CloudTrail doesn't have a concept of Backup.

**Comment by PatMyron** [comment]
True, might need to handle `N/A` as mentioned in https://github.com/SummitRoute/aws_research/issues/3
